server/routers/jobs_router.py

```
@@ -1,13 +1,240 @@
+from tqdm.auto import tqdm
+
+def get_token_api_FT(CLIENT_ID: str, CLIENT_SECRET: str, AUTH_URL: str, scope: str) -> str:
+    data = {
+        "grant_type": "client_credentials",
+        "client_id": CLIENT_ID,
+        "client_secret": CLIENT_SECRET,
+        "scope": scope,
+    }
+    params = {"realm": "/partenaire"}
+
+    resp = requests.post(AUTH_URL, data=data, params=params)
```

Ruff (0.14.13)

Probable use of `requests` call without timeout

(S113)